vue/components/ui/molecules/select/select.vue

```diff
@@ -315,13 +360,21 @@ export const Select = {
             this.openDropdown();
             this.highlightNext();
         },
-        onAltUpKey() {
+        onAltDownKey() {
             this.openDropdown();
-            this.highlight(0);
+            this.highlight(this.options.length - 1, true);
         },
-        onAltDownKey() {
+        onAltUpKey(scroll = true) {
+            this.openDropdown();
+            this.highlight(0, scroll);
+        },
+        onPageUpKey(scroll = true) {
             this.openDropdown();
-            this.highlight(this.options.length - 1);
+            this.highlight(0, scroll);
+        },
+        onPageDownKey() {
+            this.openDropdown();
+            this.highlight(this.options.length - 1, true);

```

Comment:
These behaviours are the same, the only thing that changed was the keys. Create a method for scroll to the bottom/top and add it to theses onKeys methods.